src/ansys/additive/core/parametric_study/parametric_study.py

```diff
@@ -1568,3 +1597,213 @@ def update_format(study: ParametricStudy) -> ParametricStudy:
         new_study = ParametricStudy._new(study.file_name)
         new_study._data_frame = df
         return new_study
+
+    @save_on_return
+    def add_simulations_from_data_frame(self, df: pd.DataFrame) -> list:
+        """Add simulations from an imported CSV file to the parametric study.
+
+        Parameters
+        ----------
+        df : pd.DataFrame
+            Data frame of the csv file containing simulations to be added to the parametric study.
+
+        Returns
+        -------
+        list
+            List of error messages for invalid simulations.
+        """
+
+        # check valid inputs
+        drop_indices, error_list = [], []
+        allowed_status = [
+            SimulationStatus.COMPLETED,
+            SimulationStatus.PENDING,
+            SimulationStatus.SKIP,
```

Comment:
Now I think of it, earlier I was trying to use the `add_inputs` or `add_summaries` to populate the data frame from the csv file which did not permit simulation status error. If we are appending directly to the data frame, there is no reason to avoid the errored simulations. I will update this section.